### PR TITLE
[Suggestion] Fix typing to fit ts 4 and ts 5

### PR DESCRIPTION
**Pull request by RaananW**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/dev/core/src/Engines/thinEngine.ts [hunk 1]
```diff
@@ -407,7 +407,15 @@ export class ThinEngine {
     protected _creationOptions: EngineOptions;
     protected _audioContext: Nullable<AudioContext>;
     protected _audioDestination: Nullable<AudioDestinationNode | MediaStreamAudioDestinationNode>;
-
+    public _glSRGBExtensionValues: {
```
Comment (Popov72) [inline packages/dev/core/src/Engines/thinEngine.ts:410]: I think it needs a `/** @internal */` prefix.

**Review by Popov72 — APPROVED** [review]
LGTM! Let's see what @sebavan thinks about it.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13772/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13772/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13772/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13772/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13772/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13772/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13772/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13772/merge#BCU1XR#0

File: packages/dev/core/src/Engines/thinEngine.ts (hunk 1 above)
Comment (RaananW) [reply packages/dev/core/src/Engines/thinEngine.ts:411]: Done :-)

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13772/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13772/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13772/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by RaananW** [comment]
Fixes #13771

**Pull request merged**